compiler/ml/typetexp.ml

```diff
@@ -850,7 +850,29 @@ let report_error env ppf = function
     | other_ident ->
       Format.fprintf ppf "The value %a can't be found" Printtyp.longident
         other_ident);
-    super_spellcheck ppf Env.fold_values env lid |> ignore
+    let did_spellcheck = super_spellcheck ppf Env.fold_values env lid in
+    (* For cases such as when the user refers to something that's a value with 
+      a lowercase identifier in JS but a module in ReScript.
+      
+      'Console' is a typical example, where JS is `console.log` and ReScript is `Console.log`. *)
+    (* TODO(codemods) Add codemod for refering to the module instead. *)
+    let as_module =
+      match lid with
+      | Lident name -> (
+        try
+          Some
+            (env
+            |> Env.lookup_module ~load:false
+                 (Lident (String.capitalize_ascii name)))
+        with _ -> None)
+      | _ -> None
+    in
+    match as_module with
+    | None -> ()
+    | Some module_path ->
+      Format.fprintf ppf "@,@[<v 2>@,@[%s to use the module @{<info>%a@}?@]@]"
```

Comment:
What is `"@,@[<v 2>@,@[%s` here exactly?

Reply:
OCaml format printing.

Reply:
Yeah, I meant more specially the `@,@[<` incantation.